readme.md

```diff
@@ -199,6 +199,7 @@ These are resources that are specifically useful for working with newspapers. Ma
 - [open-oni](https://github.com/open-oni/open-oni) is a friendly fork of [chronam](https://github.com/libraryofcongress/chronam) which is a webapp for viewing National Digital Newspaper Program data from the Library of Congress.
 - [docker-open-oni](https://github.com/open-oni/docker-open-oni) is a Docker friendly setup for open-oni, which will set up and configure the Web application as well as the MySQL, Solr and RAIS Image Server.
 - [ndnp_iiif](https://github.com/umd-mith/ndnp_iiif) is a Python program for turning [National Digital Newspaper Program data](https://www.loc.gov/ndnp/) into static IIIF JSON that is ready for mounting on the Web.
+- [DigiPress - The newspaper portal of the Bayerische Staatsbibliothek (Bavarian State Library)](https://digipress.digitale-sammlungen.de/) now supports IIIF - see a [sample issue](https://digipress.digitale-sammlungen.de/view/bsb00012484_00382_u001/1), [sample collection](https://api.digitale-sammlungen.de/iiif/presentation/bsb00012484/view.html), and [collection manifest](https://api.digitale-sammlungen.de/iiif/presentation/v2/bsb00012484/manifest). 
```

Comment:
The collection implementation for digipress is currently not yet finished. What is referred to as a collection in this paragraph is merely a manifest for the bound volume of newspaper issues, which does not reference the individual issues. We are planning to release proper collection manifests (newspaper -> issues) in the future, so maybe in the meantime this paragraph should only mention the issue manifests.

Reply:
@jbaiter Any update on this? One thought was that rather than the long description with multiple links here we could link to a single page. Which page would be the best to link to?

Reply:
Well, the entry page is https://digipress.digitale-sammlungen.de (which is German-only). More illustrative for IIIF purposes would probably be the View page of a single issue, which includes a link to the IIIF manifest (like https://digipress.digitale-sammlungen.de/view/bsb00012484_00584_u001/1).